refresh the network-firewalls-pfsense-snmp documentation (#2919)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-firewalls-pfsense-snmp.md

pfSense is an open source router and firewall software based on FreeBSD and

> entirely configurable throught a user-friendly web interface.

Reviewer: ```suggestion
is entirely configurable through a user-friendly web interface.
```

The Centreon Monitoring Connector *pfSense* aims to collect the status of the interfaces

> and the several number of packets per second using with the SNMP protocol.

Reviewer: ```suggestion
and the number of packets per second using with the SNMP protocol.
```

## Pack assets

### Templates

The Monitoring Connector **pfSense** brings a host template:

* **Net-FW-Pfsense-SNMP-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="Net-FW-Pfsense-SNMP-custom" label="Net-FW-Pfsense-SNMP-custom">

| Service Alias                 | Service Template                                         | Service Description                                           | Discovery  |
|:------------------------------|:---------------------------------------------------------|:--------------------------------------------------------------|:----------:|
| Bad-Offset-Packets            | Net-FW-Pfsense-Bad-Offset-Packets-SNMP-custom            | Check number of packets with bad offset.                       |            |
| Blocked-Packets-Per-Interface | Net-FW-Pfsense-Blocked-Packets-Per-Interface-SNMP-custom | Check count of packets blocked on multiple network interfaces. | X          |
| Fragment-Packets              | Net-FW-Pfsense-Fragment-Packets-SNMP-custom              | Check number of fragmented packets.                            |            |

> | Match-Packets                 | Net-FW-Pfsense-Match-Packets-SNMP-custom                 | Check number of packets that matched a filter rule.            |            |

Reviewer: ```suggestion
| Match-Packets                 | Net-FW-Pfsense-Match-Packets-SNMP-custom                 | Check number of packets that match a filter rule.            |            |
```

> | Memory-Dropped-Packets        | Net-FW-Pfsense-Memory-Dropped-Packets-SNMP-custom        | Check count of packets dropped due to memory limitations.      |            |

Reviewer: ```suggestion
| Memory-Dropped-Packets        | Net-FW-Pfsense-Memory-Dropped-Packets-SNMP-custom        | Check number of packets dropped due to memory limitations.      |            |
```
